src/Components/Web/src/WebEventDescriptor.cs

```diff
@@ -27,7 +27,7 @@ public sealed class WebEventDescriptor
         /// <summary>
         /// For framework use only.
         /// </summary>
-        public string EventArgsType { get; set; } = default!;
+        public string EventName { get; set; } = default!;
```

Comment:
nit: This is a breaking change

Reply:
Commented about this at https://github.com/dotnet/aspnetcore/pull/29993#discussion_r573866363